src/pages/UserHome.jsx

```diff
@@ -41,12 +44,27 @@ const useStyles = makeStyles((theme) => ({
   noScholarshipsBox: {
     textAlign: 'center',
   },
+  progress: {
+    display: 'block',
+  },
+  loadMoreButton: {
+    margin: theme.spacing(3, 0),
+  },
 }));
 
 export default function UserHome() {
   const classes = useStyles();
+  const [scholarships, setUserHome] = useState([]);
+  const loading = useState(true);
+  const error = useState();
```

Comment:
Neither of these seem to be working. When I go to the user home page there's no loading indicator. Neither do any errors show up when there's an actual error. I think you forgot to:
- Initialize `loading` to `false` so that a progress indicator shows up
- [Catch](https://github.com/beyondhb1079/s4us/blob/6ffe86ee03469c43bcc0abafa6ae2774c77f2188/src/pages/Scholarships.jsx#L82) any error fetching scholarships.
- [Set `loading` to `false`](https://github.com/beyondhb1079/s4us/blob/main/src/pages/Scholarships.jsx#L83) after succesfully/failing to fetch scholarships.

Reply:
Ok it looks this was fixed in the latest commit.